berkeley-function-call-leaderboard/bfcl/utils.py

```diff
@@ -198,7 +198,7 @@ def check_api_key_supplied() -> bool:
     """
     ENV_VARS = ("GEOCODE_API_KEY", "RAPID_API_KEY", "OMDB_API_KEY", "EXCHANGERATE_API_KEY")
     for var in ENV_VARS:
-        if os.getenv(var) == "":
+        if os.getenv(var) is None:
```

Comment:
1. When the user copies the `.env.example` content to `.env` without filling in the actual key (eg, `RAPID_API_KEY=`), `os.getenv("RAPID_API_KEY")` will return `""` (empty string). 
2. When the user has no `.env` file or there is no entry on the `RAPID_API_KEY` key, `os.getenv("RAPID_API_KEY")` will return `None`. 

Since both scenarios are possible, you should handle both situations.

Reply:
Thank you for pointing that out! Just fixed it